### PR TITLE
chore: update npm libraries

### PR DESCRIPTION
**Pull request by keita-determined**
## Ticket

[ET-244]

## Description

Updated libraries that dont require many code changes, so avoided updating antd, testing-library, uplot, ESLint etc

## Test Plan

- CI pass
- no regressions

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[ET-244]: https://hpe-aiatscale.atlassian.net/browse/ET-244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/components/Table/TableBatch.tsx [hunk 1]
```diff
@@ -12,20 +12,14 @@ interface Action<T> {
 
 interface Props<T> {
   actions?: Action<T>[];
-  ids?: string[];
   onAction?: (action: T) => void;
   onClear?: () => void;
   selectedRowCount?: number;
 }
 
-const defaultProps = {
```
Comment (keita-determined) [inline webui/react/src/components/Table/TableBatch.tsx:21]: to resolve `Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.`, removed `defaultProps` and use JS default parameters

File: webui/react/src/stores/cluster.tsx [hunk 2]
```diff
@@ -276,9 +276,8 @@ class ClusterStore extends PollingStore {
     )
       .then(() => {
         this.#resourcePoolBindings.update((map) =>
-          map.update(
-            resourcePool,
-            (oldWorkspaceIds) => oldWorkspaceIds?.filter((id) => !workspaceIds.includes(id)),
+          map.update(resourcePool, (oldWorkspaceIds) =>
```
Comment (keita-determined) [inline webui/react/src/stores/cluster.tsx:279]: just format change

**Comment by ashtonG** [comment]
could you capture the `humanize-duration` change in a testing plan, please?

**Comment by keita-determined** [comment]
> could you capture the `humanize-duration` change in a testing plan, please?

added in test plan

**Comment by ashtonG** [comment]
re: test plan -- remember that members of the release party should be using this test plan to ensure that the pr works. can we succinctly describe the change in logic here and find a place in the app where users can observe the change happening?

**Comment by keita-determined** [comment]
@ashtonG reverted `humanize-duration` version change not to have test code change in this PR

**Review by ashtonG — APPROVED** [review]

**Pull request merged**